src/main/java/org/hydev/mcpm/client/database/LocalPluginTracker.java

```diff
@@ -119,48 +232,35 @@ private void updateCsv(String replace, int row, int col) throws IOException {
      * PluginName2,false
      *
      * @param name Plugin name
-     */
-    
-    public void addManuallyInstalled(String name)
+     */    
+    public void setManuallyInstalled(String name)
```

Comment:
You can merge this method and `removeManuallyInstalled` together by just taking in a `boolean` for whether or not its manual.